cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,250 @@
+       <cleanlab.rank.get_label_quality_scores>`).
+
+     filter_by : {'prune_by_class', 'prune_by_noise_rate', 'both', 'confident_learning', 'predicted_neq_given',
+         'low_normalized_margin', 'low_self_confidence'}, default='prune_by_noise_rate'
+       The specific method that can be used to filter or prune examples with label issues from a dataset.
+       Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+     frac_noise : float, default=1.0
+       This will return the "top" frac_noise * num_label_issues estimated label errors, dependent on the filtering method used,
+       Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+     num_to_remove_per_class : array_like
+       This parameter is an iterable that specifies the number of mislabeled examples to return from each class.
+       Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+     min_examples_per_class : int, default=1
+       The minimum number of examples required per class to avoid flagging as label issues.
+       Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+     confident_joint : np.ndarray, optional
+       An array of shape ``(K, 2, 2)`` representing a one-vs-rest formatted confident joint.
+       Refer to documentation for this argument in :py:func:`cleanlab.multilabel_classification.filter.find_label_issues <cleanlab.multilabel_classification.filter.find_label_issues>` for details.
+
+    n_jobs : optional
+       Number of processing threads used by multiprocessing.
+       Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
+
+     verbose : optional
+       If ``True``, prints when multiprocessing happens.
+
+     Returns
+     -------
+     per_class_label_issues : list(np.ndarray)
+        By default, this is a list of length K containing the examples where each class appears incorrectly annotated.
+        ``per_class_label_issues[k]`` is a Boolean mask of the same length as the dataset,
+        where ``True`` values indicate examples where class ``k`` appears incorrectly annotated.
+        
+        For more details, refer to :py:func:`cleanlab.multilabel_classification.filter.find_label_issues <cleanlab.multilabel_classification.filter.find_label_issues>`.
+        
+        Otherwise if `return_indices_ranked_by` is not ``None``, then this method returns multiple objects.
+        The first of these objects is a list of length K whose k-th element is:
+        an ordered list of indices of examples where class k appears incorrectly annotated, sorted by the likelihood that class k is correctly annotated.
```

Comment:
@aditya1503 I wrote this part of the docstring a little weird. Please review it closely and see if you have any improvements (But don't let the docstring get too long)